AutomatedTesting/Registry/editorpreferences.setreg

```diff
@@ -1,7 +1,15 @@
 {
     "Amazon": {
         "Preferences": {
-            "EnablePrefabSystem": true
+            "EnablePrefabSystem": true,
+            "EnableEditorModeFeedback": true
```

Comment:
This is the setreg that control viewport visual feedback mechanism (blurring everything and highlighting focused instance). We shouldn't be turning this on and leave it to the viewport team to enable this when they are ready. Please remove this.

Reply:
+1